hbt/selection/default.py

```diff
@@ -78,51 +69,81 @@ def default(
     events, jet_results = self[jet_selection](events, trigger_results, lepton_results, **kwargs)
     results += jet_results
 
-    # mc-only functions
-    if self.dataset_inst.is_mc:
-        events = self[mc_weight](events, **kwargs)
+    # create process ids
+    events = self[process_ids](events, **kwargs)
```

Comment:
not a new selection, so this should probably be in the main Selectors (default and "empty"), instead of here